docs/conf.py

```diff
@@ -94,7 +94,7 @@
 ]
 templates_path = ["_templates"]
 
-nbsphinx_timeout = 360
+nbsphinx_timeout = 600
```

Comment:
Please don't increase, first you should figure out why the tutorial is timing out.